src/battle/area_kmr_part_3/area.c

```diff
@@ -0,0 +1,79 @@
+#include "common.h"
+#include "battle/battle.h"
+
+#define NAMESPACE b_area_kmr_part_3
+
+extern ActorDesc N(jr_troopa);
+extern ActorDesc N(egg_jr_troopa);
+extern ActorDesc N(para_jr_troopa);
+extern ActorDesc N(spiked_para_jr_troopa);
+extern ActorDesc N(mage_jr_troopa);
+extern ActorDesc N(final_jr_troopa);
+extern ActorDesc N(paragoomba1);
+extern ActorDesc N(paragoomba2);
+extern ActorDesc N(spiked_goomba);
+
+extern Stage N(kmr_02);
+extern Stage N(kmr_03);
+extern Stage N(kmr_04);
+extern Stage N(kmr_05);
+extern Stage N(kmr_06);
+extern Stage N(nok_01);
+extern Stage N(mim_01);
+extern Stage N(sam_01);
+extern Stage N(mac_01);
+extern Stage N(kpa_01);
+
+Formation N(formation_00) = {
+    { &N(spiked_goomba), .home = { .index = 2 }, .priority = 0 },
```

Comment:
I thought in the other areas you changed this to look like the following?
```suggestion
    { .actor = &N(spiked_goomba), .home = { .index = 2 }, .priority = 0 },
```

Reply:
Maybe I did? I didn't know it was different from the script, I'll adjust it

Reply:
As long as its consistent :)

Reply:
I'd personally like it to look like the suggested change tbh - In case there is still a decision to be made

Reply:
Alright add `.actor =` to the tool and these formations and this PR is good for me